hueplusplus/include/winHttpHandler.h

```diff
@@ -0,0 +1,62 @@
+/**
+\file winHttpHandler.h
+Copyright Notice\n
+Copyright (C) 2017  Jan Rogall		- developer\n
+Copyright (C) 2017  Moritz Wirger	- developer\n
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 3 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+**/
+
+#ifndef _WINHTTPHANDLER_H
+#define _WINHTTPHANDLER_H
+
+#include <string>
+#include <vector>
+#include <winsock2.h>
+
+#include "IHttpHandler.h"
+
+#include "json/json.h"
+
+//! Class to handle http requests and multicast requests on windows systems
+class winHttpHandler : public IHttpHandler
```

Comment:
Naming the class winHttpHandler with lowercase letter is not consistent with naming of other classes

Reply:
Changed in 88d61e3